lambdatimes/src/components/Content/Card.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
+import propTypes from 'prop-types';
 
 const Card = props => {
   return (
     <div className="card">
-      <div className="headline">{/* headline goes here */}</div>
+      <div className="headline">{props.card.headline}</div>
       <div className="author">
         <div className="img-container">
-          <img src={'' /* image source goes here */} />
+          <img src={props.card.img} alt="Image" />
```

Comment:
You want the `alt` tag to be something other than just Image or picture or something like that. Needs to be more descriptive than that